Automattic/wp-calypso
# Emails: Inbox placeholder for calypso

#### Changes proposed in this Pull Request

This pull request adds a routing and a place holder for the new upcoming feature, mailbox.

#### Testing instructions

Check that you see an empty page under: [http://calypso.localhost:3000/email/inbox](http://calypso.localhost:3000/email/inbox)

Related to {1200182182542585-as-1200979289212216}
